gast_to_code.py

```diff
@@ -15,27 +15,28 @@ def list_helper(gast_list, out_lang, csv_delimiter = ", "):
     
     return out
         
-# py_specific_helpers
-def py_logStatement(gast):
-    arg_string = gast_router(gast["args"],"py")
-    return "print(" + arg_string + ")"
-
 def py_varAssign(gast):
     value = gast_router(gast["varValue"], "py")
-    return gast["varId"] + " = " + value
-
-# js_specific_helpers
-def js_logStatement(gast):
-    arg_string = gast_router(gast["args"],"js")
-    return "console.log(" + arg_string + ")"
+    return gast_router(gast["varId"], "py") + " = " + value
 
 def js_varAssign(gast):
```

Comment:
Suggestion: rename gast in the function arg to node/root since you are operating on a node and not necessarily the whole gast. Here and everywhere.

Reply:
We will do this too once we merge.